app/(tabs)/toybox/stats/index.tsx

```diff
@@ -5,34 +5,18 @@ import { useAppLanguageContext } from "@/context/AppLanguageContext";
 import { Link } from "expo-router";
 import { useStats } from "@/hooks/useStats";
 import { StyleSheet } from "react-native";
+import { useTable } from 'react-table';
+import React from "react"
 
 function makeTableHTML(myArray: String[][]) {
```

Comment:
This function will only work on web. Will absolutely crash on Android / iOS.

Reply:
They were replaced with Themed components extending react-table